### PR TITLE
At cmd cfun

### PR DESCRIPTION
**Pull request by pielkamil**

### CODE REVIEW

File: moler/cmd/at/get_functionality_level.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+"""
+AT+CFUN
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Kamil Pielka'
+__copyright__ = 'Copyright (C) 2020, Nokia'
```
Comment (marcin-usielski) [inline moler/cmd/at/get_functionality_level.py:11]: 2022 ;)

File: moler/cmd/at/get_functionality_level.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        :return: None
+        """
+        if is_full_line:
+            try:
+                self._parse_fun_level(line)
+            except ParsingDone:
+                pass
+        return super(GetFunctionalityLevel, self).on_new_line(line, is_full_line)
+
+    _re_fun_level = re.compile(r'^\s*\+CFUN:\s*(?P<fun_level>[\d+]{1,3})\s*$')
```
Comment (marcin-usielski) [inline moler/cmd/at/get_functionality_level.py:57]: Please add comment with example of any string to match by this regex.

File: moler/cmd/at/set_functionality_level.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+"""
+AT+CFUN
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Kamil Pielka'
+__copyright__ = 'Copyright (C) 2021, Nokia'
```
Comment (marcin-usielski) [inline moler/cmd/at/set_functionality_level.py:11]: 2022 ;)

File: moler/cmd/at/set_functionality_level.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        :param rst: whether to restart UE before setting level of functionality
+                    None - there is not provide openly any value in command (AT command uses default value)
+                    0 - not reset UE before setting level of functionality (default value)
+                    1 - reset UE before setting level of functionality (level of functionality must be 1)
+        :param connection: moler connection to device
+        :param prompt: start prompt (on system where command cd starts)
+        :param newline_chars: Characters to split lines
+        :param runner: Runner to run command
+        """
+        super(SetFunctionalityLevel, self).__init__(connection, operation='execute', prompt=prompt,
```
Comment (marcin-usielski) [inline moler/cmd/at/set_functionality_level.py:44]: Please use one method of passing arguments (connection=connection).

File: moler/cmd/at/get_functionality_level.py (hunk 1 above)
Comment (pielkamil) [reply moler/cmd/at/get_functionality_level.py]: Corrected

File: moler/cmd/at/get_functionality_level.py (hunk 2 above)
Comment (pielkamil) [reply moler/cmd/at/get_functionality_level.py:58]: Example of string added

File: moler/cmd/at/set_functionality_level.py (hunk 3 above)
Comment (pielkamil) [reply moler/cmd/at/set_functionality_level.py]: Corrected

File: moler/cmd/at/set_functionality_level.py (hunk 4 above)
Comment (pielkamil) [reply moler/cmd/at/set_functionality_level.py]: Corrected

File: moler/cmd/at/get_functionality_level.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    """
+    Command to get UE's level of functionality. Example output:
+
+    +CFUN: 1
+
+    OK
+    """
+    def __init__(self, connection=None, prompt=None, newline_chars=None, runner=None):
+        """Create instance of GetFunctionalityLevel class"""
+        super(GetFunctionalityLevel, self).__init__(connection, operation='execute', prompt=prompt,
```
[the remark this answers is not in the document]
Comment (pielkamil) [reply moler/cmd/at/get_functionality_level.py]: Corrected

**Pull request merged**